{"name": "Проверка создания пользователя по surname, длина которого больше допустимого (> 255 символов)", "status": "failed", "statusDetails": {"message": "AssertionError: Код ответа приложения должен быть 400\nassert 500 == 400\n  +500\n  -400", "trace": "self = <test_api.TestApiAuthorizedUser object at 0x7f605cbf0d60>\ncreate_user_data_with_middle_name = {'email': 'nTFRs8a2mbC@mail.ru', 'middle_name': 'DonnaHarper', 'name': 'Donna', 'password': 'VvsDtrLkO', ...}\n\n    @allure.title(\"Проверка создания пользователя по surname, длина которого больше допустимого (> 255 символов)\")\n    @pytest.mark.API\n    def test_post_create_user_invalid_data_surname_length_more_than_acceptable_bug(self,\n                                                                                   create_user_data_with_middle_name):\n        \"\"\"\n         Тестирование: Добавление пользователя с неккоректными данными\n         Эндпоинт: POST http://127.0.0.1:8086/api/user\n         Предусловия: Авторизация, создание пользовательских данных, заменить surname на surname, длиной больше\n         допустимого (больше чем 255 символов)\n         Шаги:\n         1. Отправка запроса по API\n         Ожидаемый результат:\n         Код ответа - 400 (\"BAD REQUEST\") (-)\n         Пользователь НЕ добавляется в БД (+)\n         Фактический результат:\n         Код ответа - 500 (\"INTERNAL SERVER ERROR\")\n         Пользователь НЕ добавляется в БД\n         \"\"\"\n        complex_user_data = create_user_data_with_middle_name\n        complex_user_data[\"surname\"] = RandomGenerate.generate_random_user_name(256, 300)\n    \n        response = self.post_create_user(complex_user_data)\n        self.mysql.session.commit()\n        user_db = self.mysql.session.query(Users).filter_by(email=complex_user_data['email']).first()\n    \n>       assert response.status_code == 400, \"Код ответа приложения должен быть 400\"\nE       AssertionError: Код ответа приложения должен быть 400\nE       assert 500 == 400\nE         +500\nE         -400\n\ncomplex_user_data = {'email': 'nTFRs8a2mbC@mail.ru', 'middle_name': 'DonnaHarper', 'name': 'Donna', 'password': 'VvsDtrLkO', ...}\ncreate_user_data_with_middle_name = {'email': 'nTFRs8a2mbC@mail.ru', 'middle_name': 'DonnaHarper', 'name': 'Donna', 'password': 'VvsDtrLkO', ...}\nresponse   = <Response [500]>\nself       = <test_api.TestApiAuthorizedUser object at 0x7f605cbf0d60>\nuser_db    = None\n\ncode/api_myapp/test/test_api.py:512: AssertionError"}, "description": "\n         Тестирование: Добавление пользователя с неккоректными данными\n         Эндпоинт: POST http://127.0.0.1:8086/api/user\n         Предусловия: Авторизация, создание пользовательских данных, заменить surname на surname, длиной больше\n         допустимого (больше чем 255 символов)\n         Шаги:\n         1. Отправка запроса по API\n         Ожидаемый результат:\n         Код ответа - 400 (\"BAD REQUEST\") (-)\n         Пользователь НЕ добавляется в БД (+)\n         Фактический результат:\n         Код ответа - 500 (\"INTERNAL SERVER ERROR\")\n         Пользователь НЕ добавляется в БД\n         ", "steps": [{"name": "Выполняем POST запрос '/api/user'", "status": "passed", "steps": [{"name": "Выполняем запрос", "status": "passed", "parameters": [{"name": "method", "value": "'POST'"}, {"name": "location", "value": "'/api/user'"}, {"name": "headers", "value": "{'Content-Type': 'application/json'}"}, {"name": "data", "value": "None"}, {"name": "params", "value": "None"}, {"name": "json", "value": "{'username': 'sr8LRsAkxJEpE', 'password': 'VvsDtrLkO', 'email': 'nTFRs8a2mbC@mail.ru', 'name': 'Donna', 'surname': 'SBSD7QjeMHlWLo3UJtkG9Mh5vKlx5m6LNtjKzCmSzRjvPJFZ0fQoxrqxgun6c8rvPj035FPhQMPGd0AG2z9bwWczj9Uea48dYXmylv01MqT6VB5EpFbXpp71VT2h16RiqfivUiCtzCjY5yBupuf8CaAbQ2KfIDlFFK9vZoWf4CCDzONzmF18lDHLOFwZ1wfYVTlomBhQyEp8Vgq02Kb1ttnS7GD4tw3FkvQO5yPsplyDxj2qppKlV25iw05evC51unf1LXTcn66e2NvxRDOPkYPYGxSUhLdvWuwAjO', 'middle_name': 'DonnaHarper'}"}], "start": 1653171954557, "stop": 1653171954567}], "parameters": [{"name": "complex_data", "value": "{'username': 'sr8LRsAkxJEpE', 'password': 'VvsDtrLkO', 'email': 'nTFRs8a2mbC@mail.ru', 'name': 'Donna', 'surname': 'SBSD7QjeMHlWLo3UJtkG9Mh5vKlx5m6LNtjKzCmSzRjvPJFZ0fQoxrqxgun6c8rvPj035FPhQMPGd0AG2z9bwWczj9Uea48dYXmylv01MqT6VB5EpFbXpp71VT2h16RiqfivUiCtzCjY5yBupuf8CaAbQ2KfIDlFFK9vZoWf4CCDzONzmF18lDHLOFwZ1wfYVTlomBhQyEp8Vgq02Kb1ttnS7GD4tw3FkvQO5yPsplyDxj2qppKlV25iw05evC51unf1LXTcn66e2NvxRDOPkYPYGxSUhLdvWuwAjO', 'middle_name': 'DonnaHarper'}"}], "start": 1653171954557, "stop": 1653171954567}], "start": 1653171954557, "stop": 1653171954573, "uuid": "d0c728c6-59fd-42cf-9a18-a82766523f9f", "historyId": "15a6da9270f868b9563117ac7b797ff1", "testCaseId": "1e611c54609c63c56a705a39249397da", "fullName": "code.api_myapp.test.test_api.TestApiAuthorizedUser#test_post_create_user_invalid_data_surname_length_more_than_acceptable_bug", "labels": [{"name": "tag", "value": "API"}, {"name": "parentSuite", "value": "code.api_myapp.test"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApiAuthorizedUser"}, {"name": "host", "value": "91011fa0eb27"}, {"name": "thread", "value": "1-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "code.api_myapp.test.test_api"}]}